solr/solr-ref-guide/src/metrics-reporting.adoc

```diff
@@ -53,6 +53,12 @@ This registry is returned at `solr.jvm` and includes the following information.
 * System properties such as Java information, various installation directory paths, ports, and similar information. You can control what appears here by modifying `solr.xml`.
 // TODO for 7.0 fix this
 
+=== Overseer Registry
```

Comment:
We can update this doc comment now, since we are using the CoreContainer registry

Reply:
chose to leave it this way - although we're using MetricManager from CoreContainer i'm still registering these metrics under "solr.overseer."